### PR TITLE
EJ2-72228 : Hide type and upload signature dialoge in PDF Viewer

### PR DESCRIPTION
**Pull request by SaravanaPriya31**
### Description 

Hide type and upload signature dialoge in PDF Viewer

### CODE REVIEW

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/hide-type-and-uploadsignature.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+layout: post
+title: Hide the type and upload signature dialogue | Syncfusion
+description: Learn how to hide the type and upload signature dialogue in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: Hide the type and upload signature dialogue
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+# How to Hide the type and upload signature dialogue
```
Comment (florenceliliana) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/hide-type-and-uploadsignature.md:11]: # How to Hide the type and upload Signature Dialogue

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/hide-type-and-uploadsignature.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+To hide the type and upload signature dialogue in Syncfusion PDF Viewer, you can use the following code
```
Comment (florenceliliana) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/hide-type-and-uploadsignature.md:13]: You can use the following code to hide the type and upload the signature dialogue in the Syncfusion PDF Viewer.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/hide-type-and-uploadsignature.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            hideTypeSelection: true, // hide the signature field type selection dialogue
+            hideUploadSignature: true, // hide the upload signature dialogue
+            hideSaveSignature: true // hide the save signature dialogue
+        };
+    };
+</script>
+
+```
+
+In the signatureDialogSettings object, set the `hideTypeSignature` and `hideUploadSignature` properties to `true` to hide the type and upload signature dialogue, respectively. Additionally, set the `hideSaveSignature property` to `true` to hide the Save Signature button.The `displayMode `property is used to control the display mode of the signature dialog box. Setting it to `0` will hide the dialog box.
```
Comment (florenceliliana) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/hide-type-and-uploadsignature.md:40]: In the signatureDialogSettings object, set the `hideTypeSignature` and `hideUploadSignature` properties to `true` to hide the type and upload signature dialogue, respectively. Additionally, set the `hideSaveSignature property` to `true` to hide the Save Signature button. The `displayMode `property controls the display mode of the signature dialog box. Setting it to `0` will hide the dialog box.

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md:2]: Layout: Post

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: How to Customize Signature Dialog | Syncfusion
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md:3]: Title: How to Customize Signature Dialog | Syncfusion

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: How to Customize Signature Dialog | Syncfusion
+description: Learn how to customize signature dialog in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md:4]: Description: Learn how to customize signature dialog in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: How to Customize Signature Dialog | Syncfusion
+description: Learn how to customize signature dialog in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md:5]: Platform: ej2-asp-core-mvc

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: How to Customize Signature Dialog | Syncfusion
+description: Learn how to customize signature dialog in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to Customize Signature Dialog
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md:6]: Control: How to Customize Signature Dialog

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: How to Customize Signature Dialog | Syncfusion
+description: Learn how to customize signature dialog in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to Customize Signature Dialog
+publishingplatform: ##Platform_Name##
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md:7]: Publishingplatform: ##Platform_Name##

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: How to Customize Signature Dialog | Syncfusion
+description: Learn how to customize signature dialog in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to Customize Signature Dialog
+publishingplatform: ##Platform_Name##
+documentation: ug
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md:8]: Documentation: UG

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+description: Learn how to customize signature dialog in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to Customize Signature Dialog
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+# How to Customize Signature Dialog
+
+To customize the signature dialogue in Syncfusion PDF Viewer. The `signatureFieldSettings` property of the PDF Viewer instance is used to access the signature dialog settings.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md:13]: Customize the signature dialogue in Syncfusion PDF Viewer. The `signatureFieldSettings` property of the PDF Viewer instance is used to access the signature dialog settings.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+
+The `displayMode` property determines the display mode of the signature dialogue. A value of `1` indicates that the signature dialogue should be displayed in a separate window. The `hideTypeSelection` property is used to hide the signature field type selection dialogue, which allows users to choose the type of signature field they want to add. The `hideUploadSignature` property is used to hide the upload signature dialogue, which allows users to upload an image file to use as their signature. The `hideSaveSignature property` is used to hide the save signature dialogue, which allows users to save their signature for future use. The `hideDrawSignature` property is set to true, which means that the Draw Signature option will be hidden in the signature dialog.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md:15]: The `displayMode` property determines the display mode of the signature dialogue. A value of `1` indicates that the signature dialogue should be displayed in a separate window. The `hideTypeSelection` property hides the signature field type selection dialogue, allowing users to choose the type of signature field they want to add. The `hideUploadSignature` property is used to hide the upload signature dialogue, which allows users to upload an image file to use as their signature. The `hideSaveSignature` property hides the save signature dialogue, allowing users to save their signature for future use. The `hideDrawSignature` property is set to true, meaning the Draw Signature option will be hidden in the signature dialog.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+
+You can modify these properties to suit your needs and customize the signature dialogue in Syncfusion PDF Viewer accordingly.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/customize-signature-dialog.md:17]: Modify these properties to suit your needs and customize the signature dialogue in Syncfusion PDF Viewer accordingly.

**Pull request closed without merging**